go

```diff
@@ -1,9 +1,20 @@
-#! /bin/bash
+#! /bin/bash -e
 
 command_exists () {
   type "$1" &> /dev/null ;
 }
 
+run_docker() {
+  docker run \
+  -it \
+  --rm \
+  -p "8000:8000" \
+  -w "/app" \
+  -v $(pwd):/app \
+  $docker_image \
+  "${@-bash}"
+}
```

Comment:
There might be usergroup permission issue if we create files inside the docker container.
Probably need to add something like `-u $(id -u):$(id -g)` just in case.